components/org.wso2.micro.integrator.core/src/main/java/org/wso2/micro/integrator/core/internal/CoreServerInitializer.java

```diff
@@ -612,8 +582,8 @@ private void registerCarbonServlet(HttpService httpService, HttpContext defaultH
     }
 
     private void initNetworkUtils(AxisConfiguration axisConfiguration) throws AxisFault, SocketException {
```

Comment:
We can remove this method call and its usage as it relates to mgt console

Reply:
cannot remove completely since it init server hostname not only management host name